telemetry/jetbrains/src/test/resources/testResultOutput

```diff
@@ -32,12 +31,14 @@ enum class Result(
 
     DashSeparated("dash-separated"),
 
-    UnderscoreSeparated("underscore_separated");
+    UnderscoreSeparated("underscore_separated"),
+
+    Unknown("unknown");
 
     override fun toString(): String = value
 
     companion object {
-        fun from(type: Any): Result = values().first { it.value == type.toString() }
```

Comment:
Why does this no longer take any `Any`?

Reply:
Why would it take an Any? 
We forced the value to be a String, so when would we not want it to take a string in the from? 

Implicitly relying on a toString to match the value of this seems more dirty than making the call site be aware that it owns the conversion